data/transactions/logic/langspec.json

```diff
@@ -1090,7 +1588,12 @@
     {
       "Opcode": 67,
       "Name": "return",
-      "Args": "U",
+      "Args": [
+        "uint64"
+      ],
+      "Returns": [
+        "none"
+      ],
```

Comment:
I'm becoming more convinced these "none" returns need to be elided. A simpleton interpretation of this is that there is one value returned on the stack.